samples/web/asp-mvc-application/sample.md

```diff
@@ -45,13 +45,7 @@ In `AsyncPagesMvc`, open `Global.asax.cs` and see the code for the `ApplicationS
 
 snippet:ApplicationStart
 
-By calling `With()`, the code indicates to NServiceBus to scan the directory where the web application is deployed (different from non-web applications).
-
-The `.ForMvc` extension method injects `IBus` into the controllers by implementing the MVC interfaces `IDependencyResolver` and `IControllerActivator`.
-
-The NServiceBus builder registers and instantiates `IControllerActivator` so that when the controllers are requested, the NServiceBus builder has the opportunity to inject the `IBus` implementation into their `IBus` public property.
-
-Read [how the IBus is injected into the controllers](/samples/web/asp-mvc-injecting-bus/).
+Read [how the IEndpointInstance/IBus is injected into the controllers](/samples/web/asp-mvc-injecting-bus/).
```

Comment:
> Read [how the IEndpointInstance/IBus is injected into the controllers](/samples/web/asp-mvc-injecting-bus/).

How about:

For more details on how to either inject the IBus (for Versions 5 and below) or the IEndpointInstance (for Versions 6 and above) to the controllers, check the [ASP.NET MVC Sample](/samples/web/asp-mvc-injecting-bus/).